.azure-pipelines/build-job.yml

```diff
@@ -101,6 +101,10 @@ jobs:
       parameters:
         branch: ${{ parameters.branch }}
 
+  - ${{ if eq(parameters.arch, 'arm64') }}:
+    - script: sudo apt-get update && sudo apt-get -y install unzip
+      displayName: Install unzip
```

Comment:
Could you please add a couple of notes here - why it is being called specifically for arm pool?